README.md

```diff
@@ -1,22 +1,76 @@
 
-moja global welcomes a wide range of contributions as explained in [Contributing document](https://github.com/moja-global/About-moja-global/blob/master/CONTRIBUTING.md) and in the [About moja-global Wiki](https://github.com/moja-global/.github/wiki).  
+## Usage
 
-  
-## FAQ and Other Questions  
+To set up this project locally follows the below procedure:
 
-* You can find FAQs on the [Wiki](https://github.com/moja.global/.github/wiki).  
-* If you have a question about the code, submit [user feedback](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Provide-User-Feedback.md) in the relevant repository  
-* If you have a general question about a project or repository or moja global, [join moja global](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Join-moja-global.md) and 
-    * [submit a discussion](https://help.github.com/en/articles/about-team-discussions) to the project, repository or moja global [team](https://github.com/orgs/moja-global/teams)
-    * [submit a message](https://get.slack.help/hc/en-us/categories/200111606#send-messages) to the relevant channel on [moja global's Slack workspace](mojaglobal.slack.com). 
-* If you have other questions, please write to info@moja.global   
-  
+1. Clone [FLINT.Example](https://github.com/moja-global/FLINT.Example) repository. A `Dockerfile` is inside the `Docker` folder. So to build the image follow the below command:
+
+```shell
+cd Docker
+docker build --build-arg NUM_CPU=8 -t moja/flint.example:bionic .
+```
+
+It will take some time to build the image. For more information, you can follow [https://docs.moja.global/en/latest/DevelopmentSetup/docker_installation_example.html](https://docs.moja.global/en/latest/DevelopmentSetup/docker_installation_example.html).
+
+1. Clone [FLINT.UI](https://github.com/moja-global/FLINT-UI) repository and get inside the repository directory. Now you have to build a `docker-compose.yml` file so follow the below command:
+
+```shell
+docker-compose up -d
+```
+
+This will build all the docker images inside the `docker-compose.yml` file. It will take some time and after that, you can see all images list in the Docker app.
+
+<div align="center">
+<img src="assets/docker-images.jpg">
+</div>
+
+Now you can run any all the containers by clicking on the `RUN` button. You can use the command line too to run these images. All containers list will look like this in the Docker app:
+
+**NOTE:** You may encounter a problem with the `flint-ui_flint.gcbm.api` image because it is not refactored for a local environment yet.
+
+<div align="center">
+<img src="assets/docker-containers.jpg">
+</div>
+
+1. Now you can run the Vue app to get the dashboard. Go to the `flint.ui` folder of [FLINT.UI](https://github.com/moja-global/FLINT-UI) repository and run the below command:
```

Comment:
This should already be running in the container, but might still be helpful to build the Vue app locally if it's faster? We still haven't quite cracked hot reloading within the container.